src/test/java/com/automation/test/BuyPracticeTest.java

```diff
@@ -0,0 +1,84 @@
+package com.automation.test;
+
+import com.automation.test.util.BrowserManager;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.openqa.selenium.By;
+import org.openqa.selenium.JavascriptExecutor;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.interactions.Actions;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.FluentWait;
+import org.openqa.selenium.support.ui.Wait;
+
+import java.time.Duration;
+import java.util.NoSuchElementException;
+import java.util.concurrent.TimeUnit;
+
+
+public class BuyPracticeTest {
```

Comment:
can you explain to me why did you name the class like thtat

Reply:
I just put that name. I was thinking to change it for BuyClothesTest. Would It be better in that way or there is some convention to name it?

Reply:
change to BuyClothesTest